test/conformance/kuttl/mutate/clusterpolicy/cornercases/jmespath-with-special-chars/02-delete-policy.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
Do you need this step ?

Reply:
The policy remains even after the test is complete since it's namespace scoped. So I guess I need it. I can remove it if you prefer that.

Reply:
If the policy remains after the test, it could cause problems once there are lots of tests. So deleting it should avoid future problems that would be difficult to debug.